### PR TITLE
SOC-602 | Non dismissible notification when user email not confirmed

### PR DESCRIPTION
**Pull request by kvas-damian**
https://wikia-inc.atlassian.net/browse/SOC-602

### CODE REVIEW

**Comment by rafalkalinski** [comment]
LGTM

**Comment by bkoval** [comment]
LGTM

File: extensions/wikia/BannerNotifications/BannerNotificationsController.class.php [hunk 1]
```diff
@@ -269,6 +286,26 @@ public static function onSkinAfterBottomScripts( $skin, &$text ) {
 	 * @return bool
 	 */
 	public static function onBeforePageDisplay( \OutputPage $out ) {
+		global $wgUser;
+
+		if ( $wgUser->isLoggedIn() ) {
+			$message = null;
+
+			if ( empty($wgUser->getEmail() ) && empty( $wgUser->getNewEmail() ) ) {
+				$message = wfMessage('bannernotifications-no-email')->parse();
```
Comment (bkoval) [inline extensions/wikia/BannerNotifications/BannerNotificationsController.class.php]: Shouldn't there be spaces inside wfMessage brackets?

File: extensions/wikia/BannerNotifications/BannerNotificationsController.class.php [hunk 2]
```diff
@@ -269,6 +286,26 @@ public static function onSkinAfterBottomScripts( $skin, &$text ) {
...
+			} elseif ( !$wgUser->isEmailConfirmed() ) {
+				$message = wfMessage('bannernotifications-not-confirmed-email')->parse();
```
Comment (bkoval) [inline extensions/wikia/BannerNotifications/BannerNotificationsController.class.php]: Same here

**Comment by bkoval** [comment]
2 whitespacing comments, except that LGTM :hammer:

**Comment by kvas-damian** [comment]
Fixed

**Comment by bkoval** [comment]
LGTM :laughing:

**Comment by rafalkalinski** [comment]
LGTM

**Pull request merged**